lib/rubotnik/postback_dispatch.rb

```diff
@@ -36,15 +36,15 @@ def bind(regex_string, to: nil, reply_with: {})
         yield
         return
       end
-      handle_commands(to, reply_with)
+      handle_command(to, reply_with)
```

Comment:
Renamed it to keep consistent with `message_dispatch`

Reply:
cool!